### DISCUSSION TITLE
Controller Support?

### DISCUSSION THREAD

**Discussion by SecularSteve** [Neverball/neverball#381] [Ideas]
I mean, I think it's about time to give the game at least the most basic support for Dinput/Xinput/SDL2 compliant devices.

I think I'm more surprised that this was never brought to light as far as my knowledge takes me.

If there's a reason it can't happen, I'd be happy to know why.

**Comment by parasti** [comment]
Neverball has had controller support since day one. You can even use a controller to play https://play.neverball.org
